packages/universal-wallet-fastify-plugin/src/routes/base.ts

```diff
@@ -0,0 +1,69 @@
+export default (options: any) => {
+  return (fastify: any) => {
+    fastify.post(
```

Comment:
POST and `/add` are redundant. How about POST /:walletId/wallet/contents for consistency?

Reply:
I was aiming for more of a controller style interface than collection:

https://restfulapi.net/resource-naming/

we could do POST / DELETE on `wallet/contents`

Reply:
@gjgd this is related: https://stackoverflow.com/questions/299628/is-an-entity-body-allowed-for-an-http-delete-request

It does feel like wallet interfaces that are all POST / controller style might be a better starting point....

Reply:
we did this.